docs/digits_diffusion_model.md

```diff
@@ -0,0 +1,821 @@
+---
+jupytext:
```

Comment:
There should be a `formats` entry here. Did you sync the ipynb and md sources? https://jax-ai-stack.readthedocs.io/en/latest/contributing.html#adding-a-new-notebook

Reply:
This is important becuase otherwise the `md` and `ipynb` may have mismatched content.